reverse-number.js

```diff
@@ -0,0 +1,12 @@
+// Napisz funkcję reverseNumber(), która odwróci podaną liczbę (1234 => 4321)
+
+function reverseNumber(number) {
+    const newNumber = number.split("").reverse().join('');
+    return newNumber;
+
+}
+
+console.log(reverseNumber("1234")); //4321
```

Comment:
"1234" is not a number. It's a string. You have to pass a number as a parameter.